gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -821,85 +826,192 @@ local function FindCorpsePosition(corpse)
 	return false
 end
 
+local spawnPositions = {}
+for i = 0, 360, 22.5 do
+	spawnPositions[#spawnPositions + 1] = Vector(mathCos(i), mathSin(i), 0)
+end
+spawnPositions[#spawnPositions + 1] = Vector(0, 0, 1)
+
+local function MakeSpawnPositionSafe(unsafePos)
```

Comment:
Add a small comment describing the function

Reply:
Last time I did this, Histaled meant they are unnecessary and I had to remove them

Reply:
I think they are neccessary, especially if the functions are non trivial

Reply:
I think they are neccessary, escpecially when a function is complicated.

Reply:
I've already added it^^